### DISCUSSION TITLE
Experimental RTSP Audio

### DISCUSSION THREAD

**Discussion by Serverfrog** [roleoroleo/yi-hack-Allwinner-v2#259] [Q&A]
Hi,
I’m quite new in the Area of Network Cameras so I’m also not so familiar with the configuration of ONVFI and the RTSP Stream Codecs.
I use 2 h30ga at my Home now for quite the Success with Shinobi on a Raspberry Pi to Record evertyhing on my NAS. Everything is working perfectly fine… expect that I never Activated that they also should record Audio. Now I try to see how to even get the possibility to get the Audio. 
I saw that there should be an Audio Only RTSP Stream so Im thinking that the other Stream is Video+Audio. But both of them are silent. I tested with the official Yi app if I can hear something and yes, the Cameras can also record Audio. So now is my Question: what is exactly the experimental Part of the RTSP Audio Stream? Does it even work on a h30ga? Or is it more an Option from other yi-hack’s which is currently not really implemented. Or, what I’m thinking is more likely: what the hell am I doing wrong ;)

**Comment by roleoroleo** [comment]
Audio should work.
Enable it in "Configurations" web page, for example selecting "alaw".

**Reply by Serverfrog** [reply]
ok i found the Problems, first i need to restart the Camera so the setting worked.
And another thing is that the default ffmpeg doesn't support any other codec from the Camera beside aac inside the Stream Container together with the video. Need a Own Compiled version of ffmpeg for that. 
Another thing was that VLC is very picky with connection to the cameras
